CMSgov/mint-app
# [NOREF] - Fixed bullet not rendering, updated snaps

# NOREF

## Changes and Description

There was a change that added `display: flex` to a parent element of a list item, which was inherited and overrode `display:list-item` causing the bullet to not render on the first level bullet point (child bullets renders)

![Screenshot 2023-12-20 at 2 23 03 PM](https://github.com/CMSgov/mint-app/assets/95709965/cac1cfa0-5e06-4d4d-82aa-fbe0b0af1da0)

[Figma](https://www.figma.com/file/6tkIhfNfcsKEAkqXo1qvXH/MINT---Read-Only-View?type=design&node-id=3405-204354&mode=design&t=bDso5buWC9hYTemq-0)

- Removed `display-flex` to ensure bullets get rendered
- Updated snaps



## How to test this change

- Fill out some multilselect data, verify that the readonly renders the bullets correctly

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
